decko-rails/spec/controllers/card_controller_spec.rb

```diff
@@ -48,6 +53,8 @@
   end
 
   describe "#create" do
+    routes { Decko::Engine.routes }
```

Comment:
Magic so tests know about engine routes. Probably why the Decko::Engine declaration isn't in the card-gem branch. Breaks tests without this in the tests.